### ISSUE TITLE
Calendar View

### ISSUE THREAD

**Issue by ghost** [DavelBM/test.bemusical.us#12]
Shouldn't this be set by us?
![screen shot 2017-11-01 at 1 26 47 pm](https://user-images.githubusercontent.com/31675533/32296145-9e0d5d3c-bf08-11e7-9ae3-007e608e7c30.png)

**Comment by DavelBM** [comment]
@baranskiy yes we are taking it out, **take out customization of button names**
